### DISCUSSION TITLE
Overall volume wrong in case of multi-room

### DISCUSSION THREAD

**Discussion by schnillerman** [LMS-Community/slimserver#897] [General]
When a virtual "all" player is created and active, the volume is displayed e.g. as follows:

![image](https://github.com/Logitech/slimserver/assets/17252473/79aeb12a-6507-43ac-85e6-6b47d3d3aead)

The calculation of the overall volume seems to be wrong if arithmetic mean is intended?

**Comment by michaelherger** [comment]
I recommend you post to https://forums.slimdevices.com with more information about what you're using, and how you're doing things. I'm not sure whether you're dealing with the Groups Plugin, Material, or whether this indeed is a LMS issue.

**Reply by schnillerman** [reply]
Will do.

**Comment by schnillerman** [comment]
Backlink to slimdevices forums:
https://forums.slimdevices.com/forum/user-forums/logitech-media-server/1644341-overall-average-volume-wrong-in-case-of-multi-room
